source/actions/random-desktop.js

```diff
@@ -1,4 +1,5 @@
 import chalk from 'chalk';
+import Debug from 'debug';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['debug' should be listed in the project's dependencies. Run 'npm i -S debug' to add it](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5031740)